### PR TITLE
[2.3] Update labels section in README.md

### PR DESCRIPTION
**Pull request by sidolov**
### Description
Updated "Labels applied by Magento team" section in README.md file with reference to the updated documentation. 

### Fixed Issues (if relevant)

1. none
2. ...

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @sidolov. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: README.md [hunk 1]
```diff
@@ -41,20 +41,7 @@ Magento is thankful for any contribution that can improve our code base, documen
...
-| ![acknowledged](http://devdocs.magento.com/common/images/gitHub_acknowledged.png) | The Magento Team has validated the issue and an internal ticket has been created. |
-| ![in progress](http://devdocs.magento.com/common/images/github_inProgress.png) | The internal ticket is currently in progress, fix is scheduled to be delivered. |
-| ![needs update](http://devdocs.magento.com/common/images/github_needsUpdate.png) | The Magento Team needs additional information from the reporter to properly prioritize and process the issue or pull request. |
-
-To learn more about issue gate labels click [here](https://github.com/magento/magento2/wiki/Magento-Issue-Gates)
+We apply labels to public Pull Requests and Issues to help other participants retrieve additional information about current progress, component assignments, Magento release lines, and much more. Find detailed information about labels used in Magento 2 repositories in the corresponded section of the <a href="https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#labels">Code Contributions</a> guide.
```
Comment (lorikrell) [inline README.md]: I would revise the last sentence with:

`Please review the <a href="https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#labels">Code Contributions</a> guide for detailed information on labels used in Magento 2 repositories.`

**Review by lorikrell — APPROVED** [review]
After the edit, review approved from DevDocs.

**Comment by magento-engcom-team** [comment]
Hi @lorikrell, thank you for the review.
[ENGCOM-3016](https://jira.corp.magento.com/browse/ENGCOM-3016) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @sidolov. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**